### DISCUSSION TITLE
RG.Plugins.Popups 2.1.0.0

### DISCUSSION THREAD

**Discussion by mongosr2** [dansiegel/Prism.Plugin.Popups#273] [General]
I was just curious if there were plans to upgrade to RG.Plugins.Popups 2.1.0.0.  Thank you.

**Comment by dansiegel** [comment]
the master branch is currently targeting 2.1

**Reply by mongosr2** [reply]
Thank you much for the quick response.  Do you have a release planned with RG.Plugins.Popups 2.1.0.0?
